core/translate.c

```diff
@@ -606,10 +604,82 @@ translate_walk_good_state(dcontext_t *tdcontext, translate_walk_t *walk,
             (walk->in_mangle_region && translate_pc != walk->translation));
 }
 
+#ifdef AARCH64
+/* Emulate instructions in mangling epilogue. */
```

Comment:
Please put a TODO here about this not matching the existing approach and maybe going away in favor of integration with existing code.

Reply:
Done.